03-bundling/06-vite/06-typescript/README.md

```diff
@@ -272,33 +287,10 @@ Install [Node.js and npm](https://nodejs.org/en/) (14.18+ / 16+) if they are not
       plugins: [checker({ typescript: true })],
   +   build: {
   +     rollupOptions: {
-  +       plugins: [typescript({ noEmitOnError: true })],
+  +       plugins: [typescript()],
   +     },
   +   },
     });
   ```
 
-  🔎 Now try the build with and without errors to see the difference.
-
-  Also as an alternative we can just use this plugin to inform `rollup` to use the `tsconfig.json` file as the TS configuration file:
-
-  _vite.config.ts_
-
-  ```diff
-    rollupOptions: {
-  -    plugins: [typescript({ noEmitOnError: true })],
-  +    plugins: [typescript({ tsconfig: "tsconfig.json" })],
-    },
-  ```
-
-  And finally add the `noEmitOnError` setting in that file:
-
-  _tsconfig.json_
-
-  ```diff
-    "moduleResolution": "Node",
-    "noEmit": true,
-  + "noEmitOnError": true,
-    "noImplicitAny": false,
-    "noImplicitReturns": true,
-  ```
+  > By default it takes our `tsconfig.json` as reference.
```

Comment:
I've tested it won't output any bundle when TS errors exist. Even with `"noEmitOnError": false`. So I've deleted this section